Example/Tests/ObserverSetTests.swift

```diff
@@ -20,7 +20,7 @@ class ObserverSetTests: XCTestCase {
         let namedParameterObservers = ObserverSet<(name: String, count: Int)>()
         
         func testNotify() {
-            voidObservers.notify()
+            voidObservers.notify(())
```

Comment:
I hate this. Any way around it?

Reply:
I was hoping we could pass `Void` because it's a typealias but that did not work 😞

Reply:
It feels wrong to do it like this :(